docs/testing/EXPLAIN_FILES.md

```diff
@@ -1,81 +0,0 @@
-# Understanding important files in the WebApp/tests/browserstack directory
```

Comment:
Instead of completely deleting this file, can we assign someone to update it to the latest setup?

Reply:
Yeah, we could do that.